gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/PropertyValue.java

```diff
@@ -879,4 +892,15 @@ public String toString() {
       getObject().toString() :
       GConstants.NULL_STRING;
   }
+  
+  /**
+   * Throw a runtime exception if this property value can't be represented 
+   * in {@link PropertyValue#MAX_BINARY_LENGTH} bytes.
+   */
+  private void validateBytesLength() {
+    if(rawBytes != null && rawBytes.length> MAX_BINARY_LENGTH){
```

Comment:
I would provide a PR adding these two flags, but then I would need to fix 86 violations in `gradoop-common`, 950 in `gradoop-flink`, 61 in `gradoop-examples` and 84 in `gradoop-hbase`....


[INFO] Scanning for projects...
....
[INFO] --- maven-checkstyle-plugin:2.16:checkstyle (default-cli) @ gradoop-common ---
[INFO] There are **86 errors** reported by Checkstyle 6.2 with gradoop/checkstyle.xml ruleset.
[WARNING] Unable to locate Source XRef to link to - DISABLED
...                                                              
[INFO] ------------------------------------------------------------------------
[INFO] Building Gradoop Flink 0.3.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-checkstyle-plugin:2.16:checkstyle (default-cli) @ gradoop-flink ---
[INFO] There are **950 errors** reported by Checkstyle 6.2 with gradoop/checkstyle.xml ruleset.
[WARNING] Unable to locate Source XRef to link to - DISABLED
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Gradoop Examples 0.3.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-checkstyle-plugin:2.16:checkstyle (default-cli) @ gradoop-examples ---
[INFO] There are **61 errors** reported by Checkstyle 6.2 with gradoop/checkstyle.xml ruleset.
[WARNING] Unable to locate Source XRef to link to - DISABLED
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Gradoop HBase 0.3.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-checkstyle-plugin:2.16:checkstyle (default-cli) @ gradoop-hbase ---
[INFO] There are **84 errors** reported by Checkstyle 6.2 with gradoop/checkstyle.xml ruleset.
[WARNING] Unable to locate Source XRef to link to - DISABLED
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
...
[INFO] BUILD SUCCESS